### PR TITLE
Add exception parameter

### PR DESCRIPTION
**Pull request by shiwwamm**
Solves #4.
- [x] Create parameter for exceptions and set it to optional
- [x] Update docs with example and remove exceptions from the args list in the docs
- [x] The exceptions provided through the parameter should be placed after the flags passed through args during the execution in the script.

### CODE REVIEW

**Comment by shiwwamm** [comment]
@avinashupadhya99 Could you please review this.

File: README.md [hunk 1]
```diff
@@ -117,6 +117,22 @@ jobs:
...
+```yaml
+name: KubeScape-Exceptions
+on: push
+
+jobs:
+  kubescape:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: action/checkout@master
+      - uses: orrgato/Kubescape-github-action@add-exception
```
Comment (avinashupadhya99) [inline README.md]: Use `kubescape/github-action@main` for the example

**Review by avinashupadhya99 — CHANGES_REQUESTED** [review]
LGTM, just update the reference to your example in the README.

File: README.md (hunk 1 above)
Comment (shiwwamm) [reply README.md]: Updating it right now.

Comment (shiwwamm) [reply README.md]: Done. Solves #6 too.

**Review by avinashupadhya99 — APPROVED** [review]

**Comment by rtalipov** [comment]
Hi @avinashupadhya99 , @orrgato , 
What is the reason why exceptions are not supported anymore?

**Pull request merged**